HDOS-467

pka1024 opened the pull request:
feat(released timestamp):
- Added the info "released timestamp" for workflows and components in "edit workflow/component details" view.

Commit:
Show released timestamp for workflows and components

feat(released timestamp):
- Added the info "released timestamp" for workflows and components in "edit workflow/component details" view.

thatsmeta commented on frontend/src/app/components/copy-transformation-dialog/copy-transformation-dialog.component.html:
```diff
@@ -82,6 +82,16 @@ <h4 class="m-0">{{ data.title }}</h4>
     >
   </div>
+  <div>
+    <small style="font-size: 0.8em; color: gray"
```
Klasse statt inline style

Commit:
Show released timestamp for workflows and components

feat(released timestamp):
- Removed inline styles and added them by class in the stylesheet.

Commit:
Show released timestamp for workflows and components

feat(released timestamp):
- Removed some div's.
- Added display block style.
